pom.xml

```diff
@@ -37,7 +37,7 @@
     <data.pipeline.parent>system:cdap-data-pipeline[6.1.2-SNAPSHOT,7.0.0-SNAPSHOT)</data.pipeline.parent>
     <data.stream.parent>system:cdap-data-streams[6.1.2-SNAPSHOT,7.0.0-SNAPSHOT)</data.stream.parent>
     <cdap.version>6.4.0</cdap.version>
-    <hydrator.version>2.6.0</hydrator.version>
+    <hydrator.version>2.10.0-SNAPSHOT</hydrator.version>
```

Comment:
This should be 2.7.0-SNAPSHOT

Reply:
there latest changes needed her are in 2.10.0-SNAPSHOT.